roles/openshift_logging/tasks/generate_configmaps.yaml

```diff
@@ -0,0 +1,103 @@
+---
+- block:
+    - copy:
+        src: elasticsearch-logging.yml
+        dest: "{{mktemp.stdout}}/elasticsearch-logging.yml"
+      when: es_logging_contents is undefined
+
+    - copy:
+        src: elasticsearch.yml
+        dest: "{{mktemp.stdout}}/elasticsearch.yml"
+      when: es_config_contents is undefined
+
+    - lineinfile:
+        dest: "{{mktemp.stdout}}/elasticsearch.yml"
+        regexp: '^openshift\.operations\.allow_cluster_reader(.)*$'
+        line: "\nopenshift.operations.allow_cluster_reader: {{openshift_logging_es_ops_allow_cluster_reader | lower}}"
+      when: es_config_contents is undefined
```

Comment:
that makes much more sense, this was probably written before we started building templates and then never circled back to make this cleaner